metal/resource_metal_reserved_ip_block.go

```diff
@@ -49,6 +55,19 @@ func metalIPComputedFields() map[string]*schema.Schema {
 			Type:     schema.TypeBool,
 			Computed: true,
 		},
+		"wait_for_state": {
+			Type:        schema.TypeString,
+			Description: "Wait for the IP reservation block to reach a desired state on resource creation. One of: `pending`, `created`. The `created` state is default and recommended if the addresses are needed within the configuration. An error will be returned if a timeout or the `denied` state is encountered.",
+			Default:     packngo.IPReservationStateCreated,
+			Optional:    true,
+			ForceNew:    false,
```

Comment:
ForceNew=false requires the resource to have Update function defined. You can just define a dummy Update function returning Read() or set ForceNew=true like other parameters that can be updated in the UI portal have -->`Description` / `tags`

Even not required for this one-time parameter, the update method should be included in the packetngo client to match the API https://developer.equinix.com/catalog/metalv1#operation/updateIPAddress and then modify behavior here for Description/tags

Reply:
We are tracking this issue https://github.com/packethost/packngo/issues/318. When the [PR that addresses this](https://github.com/packethost/packngo/pull/319) has been merged, I'll update this PR to include the Update functionality.

Reply:
Update functionality has been added.